arrows/serialize/json/load_save_metadata.cxx

```diff
@@ -67,32 +66,50 @@ struct meta_item
       {
         const bool value = kwiver::vital::any_cast< bool > ( this->item_value );
         archive( CEREAL_NVP( value ) );
+        type = "boolean";
+      }
+      // We don't want negative ints to be serialized as large postive unsigned
+      // values since this would be confusing for external applications
+      else if ( trait.is_signed() )
+      {
+        const int value = kwiver::vital::any_cast< int > ( this->item_value );
```

Comment:
There's only a limited set of metadata tags and the only time a signed integer comes up it is an `int`. However, I I think it makes sense to update this as you suggested to avoid ugly mistakes in the future. Though I understand this does induce a slight performance penalty.